docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -138,7 +138,7 @@ or globally by setting ``ssh_args`` in ``ansible.cfg``.
 How do I get Ansible to notice a dead target in a timely manner?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` from ``ansible.cfg``. Without this option,
+You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` parameter from ``[ssh_connection]`` section of ``ansible.cfg``. Without this option,
```

Comment:
ansible.cfg is just 'one way' to do it, I would just point/link to the ssh plugin setting.

Reply:
should I use this link ? [](https://docs.ansible.com/ansible-core/devel/collections/ansible/builtin/ssh_connection.html#parameter-ssh_args )

Reply:
yep, that point at all the ways you can configure this, not just the ansible.cfg entry